terraform/services/cclf-import/main.tf

```diff
@@ -0,0 +1,86 @@
+locals {
+  full_name = "${var.app}-${var.env}-cclf-import"
+  bcda_db_envs = {
+    dev  = "dev"
+    test = "east-impl"
+    prod = "east-prod"
+  }
+  db_sg_name = {
+    bcda = "bcda-${var.env}-rds"
+  }
+  memory_size = {
+    bcda = 1024
+  }
+}
+
+data "aws_ssm_parameter" "bfd_bucket_role_arn" {
+  name = "/cclf-import/${var.app}/${var.env}/bfd-bucket-role-arn"
+}
+
+data "aws_iam_policy_document" "assume_bucket_role" {
+  statement {
+    actions   = ["sts:AssumeRole"]
+    resources = [data.aws_ssm_parameter.bfd_bucket_role_arn.value]
+  }
+}
+
+data "aws_ssm_parameter" "cclf_db_host" {
+  name = "/${var.app}/${var.env}/cclf/db-host"
+}
```

Comment:
We can remove this, the app code won't use this :)

Reply:
Removed.

Reply:
Quick check - does the app code still access the DB in some way?